platform/accessibility/common-issues/pdf/table-missing-headers.md

```diff
@@ -0,0 +1,76 @@
+# [PDF]: Table is missing headers
+
+## Table of contents
+
+  - [Descrption of the issue](#description-of-the-issue)
+  - [Definition of done](#definition-of-done)
+  - [Steps to fix the issue](#steps-to-fix-the-issue)
+    - [Define table headers in Microsoft Word](#define-table-headers-in-microsoft-word)
+    - [Define table headers in Acrobat Pro](#define-table-headers-in-acrobat-pro)
+  - [Test for compliance](#test-for-compliance)
+    - [Microsoft Word accessibility panel](#microsoft-word-accessibility-panel)
+    - [Acrobat Pro accessibility tools](#acrobat-pro-accessibility-tools)
+  - [Learn more](#learn-more)
+
+## Description of the issue
+
+- Synopsis: Make tables accessible in PDF documents by using table elements. The applicable standard from Section508.22.h dictates to "associate data and header cells for multi-dimensional data tables."
+- Severity: Critical
```

Comment:
Consider linking to severity rubric. If we're not already consistently referring to severity levels, we should do that. I think that elsewhere we use` 508-defect-0`, `508-defect-1`, etc when referring to severity levels? Maybe we should use that here as well?

Reply:
I think so, and start including document issues. I have an idea. Will add it to this PR.